### PR TITLE
015_SetAudioStreamingIndicator_param_empty_value_media_app_SDL_replie…

### PR DESCRIPTION
**Pull request by istoimenova**
Script is part of TS: #932

### CODE REVIEW

**Comment by istoimenova** [comment]
@OVikhrov, @dboltovskyi, please review.

File: test_scripts/API/SetAudioStreamingIndicator/015_ATF_N_TC_SetAudioStreamingIndicator_param_empty_value_media_app_SDL_replies_INVALID_DATA.lua [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+-- and this request is allowed by Policies
+-- SDL must NOT transfer SetAudioStreamingIndicator_request to HMI
+-- SDL must respond with result code INVALID_DATA
+--
+-- 1. Used preconditions
+-- Allow SetAudioStreamingIndicator RPC by policy
+-- Register and activate media application
+--
+-- 2. Performed steps
+-- Send SetAudioStreamingIndicator(audioStreamingIndicator = nil),
```
Comment (dboltovskyi) [inline test_scripts/API/SetAudioStreamingIndicator/015_ATF_N_TC_SetAudioStreamingIndicator_param_empty_value_media_app_SDL_replies_INVALID_DATA.lua]: @istoimenova, according to ideas file script should check empty_value for audioStreamingIndicator parameter
As far as I understand empty means blank, i.e. "", and nil means absence of parameter
We check absence of parameter in 025 script

File: test_scripts/API/SetAudioStreamingIndicator/015_ATF_N_TC_SetAudioStreamingIndicator_param_empty_value_media_app_SDL_replies_INVALID_DATA.lua [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+function Test:Precondition_ActivateApp()
+  commonSteps:ActivateAppInSpecificLevel(self, self.applications[config.application1.registerAppInterfaceParams.appName])
+  EXPECT_NOTIFICATION("OnHMIStatus", {systemContext = "MAIN", hmiLevel = "FULL"})
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+
+function Test:TestStep_SetAudioStreamingIndicator_INVALID_DATA_audioStreamingIndicator_empty()
+  local corr_id = self.mobileSession:SendRPC("SetAudioStreamingIndicator", { audioStreamingIndicator = nil })
```
Comment (dboltovskyi) [inline test_scripts/API/SetAudioStreamingIndicator/015_ATF_N_TC_SetAudioStreamingIndicator_param_empty_value_media_app_SDL_replies_INVALID_DATA.lua]: @istoimenova, according to ideas file script should check empty_value for audioStreamingIndicator parameter
As far as I understand empty means blank, i.e. "", and nil means absence of parameter
We check absence of parameter in 025 script

File: test_scripts/API/SetAudioStreamingIndicator/015_ATF_N_TC_SetAudioStreamingIndicator_param_empty_value_media_app_SDL_replies_INVALID_DATA.lua (hunk 1 above)
Comment (istoimenova) [inline test_scripts/API/SetAudioStreamingIndicator/015_ATF_N_TC_SetAudioStreamingIndicator_param_empty_value_media_app_SDL_replies_INVALID_DATA.lua]: @dboltovskyi, done in f002a9a9b435d7a8dc11907ecb01395593e26508

File: test_scripts/API/SetAudioStreamingIndicator/015_ATF_N_TC_SetAudioStreamingIndicator_param_empty_value_media_app_SDL_replies_INVALID_DATA.lua (hunk 2 above)
Comment (istoimenova) [inline test_scripts/API/SetAudioStreamingIndicator/015_ATF_N_TC_SetAudioStreamingIndicator_param_empty_value_media_app_SDL_replies_INVALID_DATA.lua]: @dboltovskyi, done in f002a9a9b435d7a8dc11907ecb01395593e26508

**Pull request merged**